frontend/src/metabase-types/api/activity.ts

```diff
@@ -48,8 +59,24 @@ export interface RecentItemsResponse {
   recent_views: RecentItem[];
 }
 
+export type RecentContexts = "selections" | "views";
+
+export interface RecentsRequest {
+  context: RecentContexts[];
+}
+
+export interface RecentsResponse {
+  recents: RecentItem[];
+}
+
 export type PopularItem = RecentItem;
 
 export interface PopularItemsResponse {
   popular_items: PopularItem[];
 }
+
+export interface CreateRecentRequest {
+  model_id: number;
+  model: ActivityModel;
+  context: "selection";
```

Comment:
I know this request will always be "selection" but can we define a type for activity context? `"view" | "selection"` ? This now comes back on RecentItem, right?

Reply:
there is a type for the GET request, those values are plural `"views" | "selections"`.

Reply:
Making sure we are all on the same page: the POST endpoint this interface is used for doesn't allow adding recent items with `context=view`, since we don't need it. Only "selection" is valid there at this time